web.go

```diff
@@ -19,7 +20,7 @@ var (
 	reApprovalURL = regexp.MustCompile(`https://[^\.]+\.(?:acm-)?certificates\.amazon\.com/approvals[^\s]+`)
 	reDomain      = regexp.MustCompile(`Domain: (.+?\.convox\.site)`)
 
-	r53 *route53.Route53
+	r53 route53iface.Route53API
```

Comment:
```suggestion
	r53 *route53iface.Route53API
```

Reply:
It's an interface so no need to take the pointer